src/playbook.yaml

```diff
@@ -109,6 +105,7 @@
 
     # Install essential R packages
     - name: Install synapser
+      become_user: ubuntu
```

Comment:
questions..
1. are you sure ubuntu user has enough privileges to install R packages?
2. why become ubuntu user only for synapser package?  why not be consistent and install all R packages with the same user?

Reply:
1. I was able to install a package as ubuntu
2. Good point, changing that...